data/remnant missions.txt

```diff
@@ -1252,3 +1252,232 @@ mission "Remnant: Expanded Horizons Quarg 3"
+			choice
+				`	(Maybe later.)`
+					goto refuse
+				`	(Yes.)`
+			`	"Excellent!" he glances at his datapad, "There are several teams working on aspects of the jump drive, and the one with the highest priority for a malfunctioning jump drive is on <planet>. I will notify them to expect you."`
+				accept
+			label refuse
+			`	He considers this, and sings, "If you wish. Our offer stands, whenever you choose to return."`
+				defer
+	on complete
+		conversation
+			`When you arrive on <planet> there is a team of engineers and mechanics waiting for you, along with a flatbed freight truck. Once you open the cargo doors they quickly free the broken jump drive from its moorings and load it delicately onto their truck. The lead engineer comes over to hand you <payment>. "Thank you for salvaging this. I suspect my compatriot who spoke to you never mentioned this, but we have a fund set aside for rewarding captains who bring in rare pieces of alien technology. If you find any more of these drives, there is sure to be a research team somewhere that is willing to reward you for it."`
+			choice
+				`	"Aren't you going to replace it with a hyperdrive?"`
+					goto hyperdrive
+				`	"I will keep a lookout for more."`
+			label hyperdrive
+			`	The engineer looks at you incredulously. "You mean you were actually depending on this drive?" He shakes his head in disbelief. "You are lucky you managed to get this far, then. It is surprising that they exist at all, let alone still work properly. If you need a replacement drive, there are plenty available in the outfitter."`
+			`	"Next time you find one of these drives, just check with starport director. They will know where it will need to go." He turns and heads off with the rest of the team, turning briefly to give you friendly wave as they disembark.`
+		outfit "Jump Drive (Broken)" -1
+		"remnant: broken jump drive count" ++
+		payment 500000
+		"reputation: Remnant" ++
+
+
+
+mission "Remnant: Broken Jump Drive 2"
+	name "Broken Jump Drive Delivery"
+	description "Please deliver a broken jump drive to the researchers on <planet>."
+	to offer
+		has "Remnant: Broken Jump Drive 1: done"
+	source
+		government "Remnant"
+	destination
+		government "Remnant"
+	on offer
+		require "Jump Drive (Broken)"
+		conversation
+			`Remembering the advice given by the engineer who paid you for the broken jump drive, you stop by the starport director's desk. Information must flow quickly amongst the Remnant, because by the time you arrive at the director's desk she already has the information ready for you.`
```

Comment:
Capitalize "director" for consistency.

Reply:
Hmmm... I'm not sure about that. There's already a bunch of references to the directors and director's desk in the learning sign missions, and they're all lower case. So for consistency, I've dropped them all down to lower case.